src/zc/buildout/buildout.py

```diff
@@ -384,7 +384,12 @@ def __init__(self, config_file, cloptions,
 
         download_cache = options.get('download-cache')
         extends_cache = options.get('extends-cache')
+
+        if bool_option(options, 'abi-tag-eggs', 'false'):
+            from zc.buildout.pep425tags import get_abi_tag
+            options['eggs-directory'] += '-' + get_abi_tag()
```

Comment:
Ahh, I guess I was combining what the wheel spec refers to as the "python tag" with the "abi tag". A `foo-1.0-py2.py3-none-any.whl` wheel has an ABI tag of "none" and a python tag of "py2.py3".